internal/services/codesigning/client/client.go

```diff
@@ -0,0 +1,27 @@
+package client
+
+import (
+	"fmt"
+
+	codesigning_v2024_09_30_preview "github.com/hashicorp/go-azure-sdk/resource-manager/codesigning/2024-09-30-preview"
+
+	"github.com/hashicorp/go-azure-sdk/sdk/client/resourcemanager"
+	"github.com/hashicorp/terraform-provider-azurerm/internal/common"
+)
+
+type Client struct {
+	V20240930previewClient codesigning_v2024_09_30_preview.Client
```

Comment:
Since this is the only client, can we just name it as `Client` or something?

Reply:
sure, updated